mta/tests/migration/controls/test_business_services.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_description, generate_name
+
+
+@pytest.mark.business
+def test_create_business_service(business_service, stakeholder, get_api, delete_api, update_api):
+    # get business_service_from_db
+    business_service_from_db = get_api.businessservices_id_get(business_service.id)
+
+    # Assertion checking the created business services's name
+    check.equal(business_service_from_db.name, business_service.name)
```

Comment:
Please check the entire objects:
`check.equal(business_service_from_db, business_service)`
If fails, we can check the name only

Reply:
Tried with the entire object and the TC fails https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/view/MTA/job/mta/job/mta-api-tests-runner/5/ 
So added the check with name `check.equal(business_service_from_db.name, business_service.name)`

Reply:
Sure. Will do